types.d.ts

```diff
@@ -79,12 +79,8 @@ import {
 	WithStatement,
 	YieldExpression
 } from "estree";
-import { JSONSchema4, JSONSchema6, JSONSchema7 } from "json-schema";
-import { default as ValidationError } from "schema-utils/declarations/ValidationError";
-import {
-	Extend,
-	ValidationErrorConfiguration
-} from "schema-utils/declarations/validate";
+import { ValidationError, validate as validateFunction } from "schema-utils";
+import { ValidationErrorConfiguration } from "schema-utils/declarations/validate";
```

Comment:
@alexander-akait This doesn't seem to be exported from the root package. It would help if you could export it from there.